libevmasm/Assembly.cpp

```diff
@@ -834,6 +862,63 @@ std::map<u256, u256> const& Assembly::optimiseInternal(
 	return *m_tagReplacements;
 }
 
+namespace
+{
+uint16_t calcMaxStackHeight(std::vector<AssemblyItem> const& _items, uint16_t _args)
```

Comment:
```suggestion
uint16_t calculateMaxStackHeight(std::vector<AssemblyItem> const& _items, uint16_t _args)
```

This isn't rust, we don't have to use abbreviations.

Reply:
Changed.